### DISCUSSION TITLE
Create a new item from putListingItem

### DISCUSSION THREAD

**Discussion by Daniel-01-10** [amzn/selling-partner-api-models#3066] [General]
How about good day colleagues.!!!
I hope you can help me. :)
My problem is that I am trying to create a new document using putListingsItem giving it the productType and Required Attributes.
The response I get is Accepted but when I try to see if my Item was published in the Amazon Seller Central listing it doesn't show up.
What would I be missing or what am I doing wrong?
I hope you can help me, I would really appreciate it
![captura](https://user-images.githubusercontent.com/34693043/191611100-ca4e4d9c-68b5-4a2a-bd3e-42140e4eadab.png)

**Comment by Vickyvignesh-01** [comment]
Hi @Daniel-01-10 i got the same issue how you rectified it

**Comment by weilinggu** [comment]
Listing submission is accepted for processing does not mean it has passed all the checks in the system. The recommendation is to listen to LISTINGS_ITEM_STATUS_CHANGE notification.
Amazon sends a LISTINGS_ITEM_STATUS_CHANGE notification when the status of a listing item changes for a selling partner. Notifications are published when a listings item is created, deleted, or its buyability changes. Subscribe to LISTINGS_ITEM_STATUS_CHANGE notification to get notified when listings are created successfully.
Check the [workflow guide](https://developer-docs.amazon.com/sp-api/docs/building-listings-management-workflows-guide#list-an-item-that-does-not-already-exist-in-the-amazon-catalog) for more details.
